### ISSUE TITLE
System.ArgumentNullException: Value cannot be null

### ISSUE THREAD

**Issue by thomasmstein** [andrew-schofield/keepass2-haveibeenpwned#86]
Hi Andrew,
Using version 1.3.4 (KP 2.43) I get this 'System.ArgumentNullException: Value cannot be null' error.

**I have checked the following options and combinations:**

```
--------------------------------------------------------------------------------
Check ... 'site/service'	
UNTICK	Only checks entries that have not been changed since the breach	-> OK
TICK	"								-> OK
--------------------------------------------------------------------------------
Check ...'username'		
UNTICK	Only checks entries that have not been changed since the breach	-> ERROR
TICK	"								-> OK
--------------------------------------------------------------------------------
Check ...'password'							-> OK
--------------------------------------------------------------------------------
Check ...'all'			

UNTICK	Only checks entries that have not been changed since the breach	-> ERROR			
TICK	All supported breaches						-> ERROR

TICK	Only checks entries that have not been changed since the breach	-> OK
TICK	All supported breaches						-> OK
---------------------------------------------------------------------------------
```
**Exception Text:**
************** Exception Text **************
System.ArgumentNullException: Value cannot be null.
Parameter name: value
   at System.String.Join(String separator, String[] value)
   at HaveIBeenPwned.UI.BreachedEntriesDialog.AddBreaches(IList`1 breaches)
   at HaveIBeenPwned.HaveIBeenPwnedExt.<CheckBreach>d__39.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at HaveIBeenPwned.HaveIBeenPwnedExt.<Entries_CheckAll>d__2e.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

![hibp_checker_error_02](https://user-images.githubusercontent.com/17682916/65844250-bdb75480-e378-11e9-8f90-86090dbe56e4.png)

Thanks
Thomas

**Comment by andrew-schofield** [comment]
Confirmed

**Comment by andrew-schofield** [comment]
OK, I've worked out what the issue is, it should be fixed soon. The plugin wasn't asking for the full breach model anymore, which meant the results dialog wasn't able to fill in most of the fields, and crashed because the breach class list was empty.

**Comment by andrew-schofield** [comment]
Fixed in v1.3.5
